examples/face_detection_yolov8onnx_example/README.md

```diff
@@ -0,0 +1,10 @@
+# tract onnx face detection demo yolov8
+face deteciton demo using yolov8 , using models converted from [derronqi's repo](https://github.com/derronqi/yolov8-face)
+
+# getting the models
+1. you can get the model and convert them yourself from [here](https://github.com/derronqi/yolov8-face) , you can follow conversion instructions [here](https://docs.ultralytics.com/integrations/onnx/)
+2. you can get it preconverted from
+[google drive](https://drive.google.com/file/d/1PYAG1ypAuwh_rDROaUF0OdLmBqOefBGL/view?usp=sharing)

```

Comment:
Since we have a way to download the model, can we make a `ci.sh` that will check our example stays in shape ? See other examples for inspiration...